site/app/controllers/OfficeHoursQueueController.php

```diff
@@ -55,11 +55,11 @@ public function openQueue() {
 
         $re = '/^[\sa-zA-Z0-9_\-]+$/m';
         preg_match_all($re, $queue_code, $matches_code, PREG_SET_ORDER, 0);
-        $matches_token = 1;
+        $matches_token = [];
         if ($token !== null) {
             preg_match_all($re, $token, $matches_token, PREG_SET_ORDER, 0);
         }
-        if (count($matches_code) !== 1 || count($matches_token) !== 1) {
+        if (count($matches_code) !== 1 && count($matches_token) !== 1) {
```

Comment:
I don't think this should have changed?

Reply:
Yeah, that was a typo and I spot it just before you commented on it, ha ha.